src/hdf5/HDF5IO.cpp

```diff
@@ -48,15 +49,16 @@ Status HDF5IO::open(bool newfile)
   if (opened)
     return Status::Failure;
 
-  FileAccPropList props = FileAccPropList::DEFAULT;
+  FileAccPropList fapl = FileAccPropList::DEFAULT;
+  H5Pset_libver_bounds(fapl.getId(), H5F_LIBVER_LATEST, H5F_LIBVER_LATEST);
```

Comment:
What is the minimum version of HDF5 that we require for this?

Reply:
I believe HDF5 1.10